python/dgl/homophily.py

```diff
@@ -0,0 +1,57 @@
+"""Utils for tacking graph homophily and heterophily"""
+from . import backend as F, function as fn
+
+__all__ = ["node_homophily"]
+
+
+def node_homophily(graph, y):
+    r"""Homophily measure from `Geom-GCN: Geometric Graph Convolutional Networks
+    <https://arxiv.org/abs/2002.05287>`__
+
+    We follow the practice of a later paper `Large Scale Learning on
```

Comment:
avoid using `later`, `new` in code, the code will be here for long, but the paper will not be new any more :)

Reply:
It emphasizes the fact that the term was not introduced in the original paper.